src/pages/mods/index.tsx

```diff
@@ -21,26 +21,69 @@ const DEFAULT_PAGE_SIZE_INDEX = 1;
 
 
 
-
 const useStyles = createStyles(
     (theme) => ({
+        table: {
+            "&&&& table": {
+                borderSpacing: "0 20px",
+            },
+        },
         pageTitle: {
             color: "white",
             fontSize: "35px",
             textAlign: "center",
         },
         modCell: {
-            //double ampersand to increase selectivity of class to ensure it overrides any other css
-            "&&": {
+            //4 ampersands to increase selectivity of class to ensure it overrides any other css
+            "&&&&": {
                 /* top | left and right | bottom */
                 padding: `${theme.spacing.xl} ${theme.spacing.xl} ${theme.spacing.xl}`,
+                backgroundColor: "#e1e1e2",
+                color: theme.black,
+                borderWidth: 0,
+                fontWeight: "bold",
             },
         },
+        expandedRow: {
+            // We move the expanded mod up to make it look like a single row.
+            "&&& + tr + tr": {
```

Comment:
Styling the table is a bit tricky - margin/padding don't work on table rows, we don't set the `<td>` ourselves so we can't create an inner child and set padding on `<td>`. I went with setting `border-spacing` on the table which separates all rows equally, and then I shift the expanded mod up to make it look like a single row.

Let me know if there is a better way and I'll change this.

Reference: https://stackoverflow.com/questions/351058/space-between-two-rows-in-a-table

Reply:
Mantine DataTable exposes a [styles api](https://icflorescu.github.io/mantine-datatable/examples/additional-styling). It's been a while since I looked through it fully, but iirc the [multiple class names](https://icflorescu.github.io/mantine-datatable/examples/additional-styling#with-multiple-class-names) and [row styling](https://icflorescu.github.io/mantine-datatable/examples/additional-styling#row-styling) sections should have what you need.

Reply:
I don't think this allows styling the expanded row. I can shift the translate to the `ExpandedMod` file, but the `translate` might be a bit confusing without the context that it is used in the mods page. I could also change `ExpandedMod` to take classes as a prop, and pass the `translate` there.

Reply:
Well the ExpandedMod component is only planned to be used on the mods page, so we can just put all of its styling in its file. If we wind up reusing it down the line we can refactor it then